Add missing return type hints to the methods of the `SentryExtension` class (#563)
getsentry/sentry-symfony: CHANGELOG.md

## Unreleased

> - Add typehint to `SentryExtension` dependency injection (Symfony 5.4).

Reviewer: ```suggestion
- Add return typehints to `SentryExtension` dependency injection to prepare for Symfony 6. (#563)
```

> - Add return typehints to `SentryExtension` dependency injection to prepare for Symfony 6. (#563)

Reviewer 2: ```suggestion
- Add return typehints to the methods of the `SentryExtension` class to prepare for Symfony 6 (#563)
```
